Resources/Prototypes/DeltaV/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -119,3 +119,17 @@
   - type: TemperatureProtection
     coefficient: 0.1
 
+- type: entity
+  parent: ClothingOuterStorageFoldableBase
+  id: ClothingOuterCoatLabPharma
+  name: pharmacist lab coat
+  description: A suit that protects against minor chemical spills. Has an orange and blue stripe on the shoulder and arm.
+  components:
+  - type: Sprite
+    sprite: DeltaV/Clothing/OuterClothing/Coats/labcoat_pharmacist.rsi
+  - type: Clothing
+    sprite: DeltaV/Clothing/OuterClothing/Coats/labcoat_pharmacist.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Caustic: 0.75
```

Comment:
Wow, that is a lot! Is this what Chemists get from their labcoats?

Reply:
This is indeed what they get from their labcoats.